perception/lidar_centerpoint/config/detection_class_adapter.param.yaml

```diff
@@ -0,0 +1,38 @@
+/**:
```

Comment:
The `detection_class_adapter.param.yaml` file should be deleted.

Reply:
Done